crates/db_schema/src/impls/moderator.rs

```diff
@@ -168,6 +168,30 @@ impl Crud for ModBan {
   }
 }
 
+impl Crud for ModHideCommunity {
```

Comment:
@dessalines in the previous code review you said it was missing admin_id around this part. What did you mean by that?